eogrow/pipelines/rasterize.py

```diff
@@ -100,9 +100,15 @@ def __init__(self, *args: Any, **kwargs: Any):
         super().__init__(*args, **kwargs)
 
         self.filename: Optional[str] = None
+        assert len(set([c.output_feature[0].is_temporal() for c in self.config.columns])) == 1, \
```

Comment:
use `any` instead of checking length. Also we try to raise `ValueError` instead of asserts (for error catching purposes)

Reply:
Also you can remove an unnecessary list there and make a set directly from generator:

```Python
set(c.output_feature[0].is_temporal() for c in self.config.columns)
```

Reply:
as discussed privately `all` and `any` get a bit long because they just all need to be equal.
So i just suggest to use set comprehension, i.e.
```python
 len({c.output_feature[0].is_temporal() for c in self.config.columns}) == 1
 ```

Reply:
Changed.